envoy.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/google/shlex"
+)
+
+func runEnvoy(args string) error {
+	argParts, err := shlex.Split(args)
+	if err != nil {
+		return err
+	}
+
+	envoyCommand := exec.Command("envoy", argParts...)
```

Comment:
Do we need to pass signals from the parent xds process into envoy? Not sure if there's a signal to handle like, graceful shutdown or what we do here, but worth noting that we're not passing any back so xds is now going to absorb it all.